Makefile

```diff
@@ -3,4 +3,23 @@ run:
 
 unit-test:
 	ginkgo -r
-	
\ No newline at end of file
+	
+start-dev-db-linux:
+	docker run --name mysql-ec-dev -d \
+        -p 3306:3306 \
+        -e MYSQL_ROOT_PASSWORD=P@ssw0rd \
+        --restart unless-stopped \
+        -v ${PWD}/database:/docker-entrypoint-initdb.d \
+        mysql:latest
+
+#not try yet
+start-dev-db-window:
+	docker run --name mysql-ec-dev -d \
+        -p 3306:3306 \
+        -e MYSQL_ROOT_PASSWORD=P@ssw0rd \
+        --restart unless-stopped \
+        -v %cd%/database:/docker-entrypoint-initdb.d \
+        mysql:latest
```

Comment:
docker start mysql-ec-dev || docker run ......
Using this command will start the existed mysql and also create one if not exists